docs/maps/maps-getting-started.asciidoc

```diff
@@ -0,0 +1,179 @@
+[[maps-getting-started]]
+== Getting started with Maps
+
+You work with *Maps* by adding layers. The data for a layer can come from 
+sources such as {es} documents, vector sources, tile map services, web map 
+services, and more. You can symbolize the data in different ways. 
+For example, you might show which airports have the longest flight 
+delays by using circles from small to big. Or, 
+you might show the amount of web log traffic by shading countries from 
+light to dark. 
+
+[role="screenshot"]
+image::maps/images/sample_data_web_logs.png[]
+
+Your best bet to getting started with *Maps* is to add one of 
```

Comment:
Just ran across this docs change that added a section for adding sample data https://github.com/elastic/kibana/pull/32244

I think we could remove this paragraph and the sample data instructions and instead just link to the add sample data page. What do you think?

Maybe change the below section to something shorter like ...

```
Your best bet to getting started with *Maps* is to add one of the {kibana-ref}/add-sample-data.html[sample data] sets that ship with {kib}. 

You'll re-create the *[Logs] Total Requests and Bytes* map in this tutorial.
Check out each of the four layers of the map. As you move your mouse over the map,
note how the tooltips show property values.

```

Reply:
@nreese Check out how I rewrote this section.  Also, I didn't link to the "add sample data" doc yet because the doc is not in this branch.